plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -136,7 +136,7 @@ def reset_port(self, port_name, port_guid):
 
         # Perform reset
         ret = self.ufm_client.reset_port(port_name, port_guid)
-        if not ret or ret.status_code != http.HTTPStatus.OK:
+        if not ret or ret.status_code not in (http.HTTPStatus.OK, http.HTTPStatus.ACCEPTED):
             self.logger.warning("Failed resetting port: %s... status_code= %s", port_name, ret.status_code)
```

Comment:
why do you need 2 status code and not a single one?

Reply:
No necessary.